priv/repo/migrations/20190606141247_create_attendees-table.exs

```diff
@@ -0,0 +1,16 @@
+defmodule :"Elixir.DevfestRegistrationPortal.Repo.Migrations.CreateAttendees-table" do
```

Comment:
Just asking why is this module name in double quotes,  :curious:

Reply:
it is coming by default